charts/datadog/values.yaml

```diff
@@ -1839,7 +1839,7 @@ kube-state-metrics:
   # # kube-state-metrics.image -- Override default image information for the kube-state-metrics container.
   # image:
   #  # kube-state-metrics.repository -- Override default image registry for the kube-state-metrics container.
-  #  repository: k8s.gcr.io/kube-state-metrics/kube-state-metrics
+  #  repository: registry.k8s.io/kube-state-metrics/kube-state-metrics
```

Comment:
the repository was already configured to use the new registry here https://github.com/DataDog/helm-charts/pull/990/files#diff-2fcec98016e714405bc936fd949012c2e72f23c20a1a94acf695a9cd3e29d75fR1809-R1811

however we can aggregate the comment and override in the same place. Would you update your PR to consolidate it?